src/web/index.html

```diff
@@ -4,7 +4,7 @@
   <title>TEAMMATES - Online Peer Feedback/Evaluation System for Student Team Projects</title>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <meta http-equiv="Content-Security-Policy" content="default-src 'none'; script-src 'self' 'unsafe-eval'; style-src 'self' 'unsafe-inline'; frame-src 'self' docs.google.com; img-src 'self' data: http: https:; font-src 'self'; connect-src 'self' ws: http://localhost:*; form-action 'none'; base-uri 'self'">
+  <meta http-equiv="Content-Security-Policy" content="default-src 'none'; script-src 'self' 'unsafe-eval' https://www.google.com/recaptcha/api.js https://www.gstatic.com/recaptcha/api2/; style-src 'self' 'unsafe-inline'; frame-src 'self' docs.google.com https://www.google.com/; img-src 'self' data: http: https:; font-src 'self'; connect-src 'self' ws: http://localhost:*; form-action 'none'; base-uri 'self'">
```

Comment:
- Do you need both values now that you're using `ngx-captcha`?
- You need to change `teammates.ui.webapi.action.WebPageServlet` as well

Reply:
I added them through trial and error by looking at the console error messages. 
From what I can tell, `ngx-captcha` calls the `recaptcha api`, which makes an internal request to `gstatic`, which makes a requests to `google`. I can look into it deeper if needed.

<img width="479" alt="2019-02-21_17h43_28" src="https://user-images.githubusercontent.com/29021586/53159451-e0bea080-3600-11e9-9c28-a69c367e4ce8.png">
<img width="479" alt="2019-02-21_17h43_55" src="https://user-images.githubusercontent.com/29021586/53159455-e1efcd80-3600-11e9-8bf7-d33c29342d45.png">
<img width="477" alt="2019-02-21_17h44_22" src="https://user-images.githubusercontent.com/29021586/53159459-e3b99100-3600-11e9-8f98-a50e1d873667.png">

Reply:
The [official documentation](https://developers.google.com/recaptcha/docs/faq#im-using-content-security-policy-csp-on-my-website-how-can-i-configure-it-to-work-with-recaptcha) says that we need all 3 URLs.
I'll update `frame-src` to be more specific